ops/scripts/ci-versions.js

```diff
@@ -6,6 +6,7 @@ const os = require('os')
 data = process.argv[2]
 data = JSON.parse(data)
 
+process.stdout.write(`::set-output name=fault-detector::0.0.0-20230123180301` + os.EOL)
```

Comment:
What is this?

Reply:
Ah yeah something is broken here and means we can't release canary versions of the fault detector or any other services that are marked as `private` in their `package.json` files.

Reply:
This was a hack to get the fault-detector to release properly. Will remove but going to create an issue.